src/lib.rs

```diff
@@ -108,18 +217,23 @@ impl Client {
         }
     }
 
-    /// Retrieves the full set of all cached claims in the lattice by getting a response from the first
-    /// host that answers this query
+    /// Retrieves the full set of all cached claims in the lattice. If a suitable key-value bucket for metadata
+    /// was discovered at client creation time, then that bucket will be queried directly for the claims. If not,
+    /// then the claims will be queried by issuing a request on a queue-subscribed topic to the listening hosts.    
     #[instrument(level = "debug", skip_all)]
     pub async fn get_claims(&self) -> Result<GetClaimsResponse> {
-        let subject = broker::queries::claims(&self.topic_prefix, &self.ns_prefix);
-        debug!("get_claims:request {}", &subject);
-        match self.request_timeout(subject, vec![], self.timeout).await {
-            Ok(msg) => {
-                let list: GetClaimsResponse = json_deserialize(&msg.payload)?;
-                Ok(list)
+        if let Some(ref store) = self.kvstore {
+            kv::get_claims(store).await
+        } else {
```

Comment:
Should there be a deprecation notice here? Are we planning on removing this in a future version?

Reply:
we might remove it in the future but the client really has no good way of reliably emitting a deprecation notice because not all of the library's consumers are using a logger